### PR TITLE
feat: Component Style Updates from Audit

### PR DESCRIPTION
**Pull request by josh-bagwell**
## Summary

Addresses discrepancies that were found in component audit between Figma and CK.

Fixes: #2477 

Needed to make style update to better align with Figma and design. 

## Release Category
Components

### Release Note
Style updates will have minor visual changes.

---

## Checklist

- [ ] MDX documentation adheres to Canvas Kit's [Documentation Guidelines](https://workday.github.io/canvas-kit/?path=/docs/guides-documentation-guidelines--page)
- [ ] Label `ready for review` has been added to PR

## Where Should the Reviewer Start?

`/modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx`

### CODE REVIEW

**Comment by josh-bagwell** [comment]
Had to adjust Popup example for visual tests since there was an adjustment made to `CardBody` and `CardHeading` sizing.

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 1]
```diff
@@ -15,10 +18,11 @@ any questions.
   - [Text Area Preview](#text-input-preview)
   - [Label Text](#label-text)
 - [Component Updates](#component-updates)
+  - [Style Updates](#style-updates)
   - [Card](#card)
   - [Count Badge](#count-badge)
   - [Text](#text)
-  - [Form Field Preview](#form-field-preview)
+	- [FormField Preview](#form-field-preview)
```
Comment (mannycarrera4) [inline modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:25]: this seems off? also add the space back in

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 2]
```diff
@@ -136,8 +140,291 @@ instead.
...
+			<Table.Cell backgroundColor='soap600'>
+				soap600
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor='licorice100'>
+				licorice100
+			</Table.Cell>
+		</Table.Row>
+  </Table.Body>
+</Table>
+
```
Comment (mannycarrera4) [inline modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:423]: Anything special you wanna call out here? The disabled delete button looks like the color is solid but we actually add an opacity now to the base color right? Might be worth adding as a note?

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 3]
```diff
@@ -136,8 +140,291 @@ instead.
 
 ## Component Updates
 
+### Style Updates
+
+We've made some small styling updates to: `Table`, `Checkbox`, `Radio`, `SecondaryButton`, `TertiaryButton`, `DeleteButton`, `TextInput` and `TextArea`.
+
+These changes are to address discrepancies that were found between Canvas Kit Design Specs and Canvas Kit.
+
+**PR:** [#2485](https://github.com/Workday/canvas-kit/pull/2485)
```
Comment (mannycarrera4) [inline modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:149]: this line should move under the title `Styles Updates`

Comment (mannycarrera4) [reply modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:149]: and maybe `Component Style Updates`

File: modules/react/button/lib/DeleteButton.tsx [hunk 4]
```diff
@@ -41,10 +41,10 @@ const deleteStyles = createStyles({
     [buttonVars.active.icon]: brand.error.accent,
   },
   '&:disabled, &:active:disabled, &:focus:disabled, &:hover:disabled': {
-    [buttonVars.disabled.background]: brand.error.light,
+    [buttonVars.disabled.background]: brand.error.base,
     [buttonVars.disabled.label]: brand.error.accent,
     [buttonVars.disabled.icon]: brand.error.accent,
-    opacity: 1,
+    [buttonVars.disabled.opacity]: '0.4',
```
Comment (mannycarrera4) [inline modules/react/button/lib/DeleteButton.tsx:47]: yeah call this out in the upgrade guide

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx (hunk 1 above)
Comment (josh-bagwell) [reply modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:25]: Agreed, addressed.

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx (hunk 2 above)
Comment (josh-bagwell) [reply modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:423]: Yeah definitely. I will address the opacity portion.

File: modules/react/button/lib/DeleteButton.tsx (hunk 4 above)
Comment (josh-bagwell) [reply modules/react/button/lib/DeleteButton.tsx:47]: Agreed

Comment (mannycarrera4) [reply modules/react/button/lib/DeleteButton.tsx:47]: I think this still needs to be called out in the upgrade guide

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 5]
```diff
@@ -15,10 +18,11 @@ any questions.
   - [Text Area Preview](#text-input-preview)
   - [Label Text](#label-text)
 - [Component Updates](#component-updates)
+  - [Style Updates](#style-updates)
```
Comment (mannycarrera4) [inline modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:21]:
```suggestion
  - [Component Style Updates](#component-style-updates)
```

**Review by mannycarrera4 — APPROVED** [review]

**Pull request merged**